README.md

```diff
@@ -53,7 +54,8 @@ gulp.task("publish", function () {
                     // use gzip for assets that benefit from it
                     gzip: true,
                     // cache static assets for 20 years
-                    cacheTime: 630720000
+                    cacheTime: 216000                     
```

Comment:
missing comma - also why is this number different from the shared cache time? the sharedCacheTime could also use an explanatory comment